soc/arm/nuvoton_npcx/npcx7/Kconfig.series

```diff
@@ -0,0 +1,16 @@
+# Nuvoton Cortex-M4 Embedded Controller NPCX7 series
+
+# Copyright (c) 2020 Nuvoton Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_NPCX7
+	bool "Nuvoton NPCX7 Series"
+	select ARM
+	select CPU_CORTEX_M4
+	select CPU_CORTEX_M_HAS_DWT
+	select CPU_HAS_FPU
+	select CPU_HAS_ARM_MPU
+	select CORTEX_M_SYSTICK
```

Comment:
It is not ideal to select CORTEX_M_SYSTICK here - this is a "user" configuration option not CPU capability.

The capability option is CPU_CORTEX_M_HAS_SYSTICK, but this is not needed for cortex-m4